skbio/sequence/_sequence.py

```diff
@@ -1302,16 +1432,21 @@ def _copy(self, deep, memo):
         bytes_ = np.copy(self._bytes)
 
         seq_copy = self._constructor(sequence=bytes_, metadata=None,
-                                     positional_metadata=None)
+                                     positional_metadata=None,
+                                     interval_metadata=None)
 
         if deep:
             seq_copy._metadata = MetadataMixin._deepcopy_(self, memo)
             seq_copy._positional_metadata = \
                 PositionalMetadataMixin._deepcopy_(self, memo)
+            seq_copy._interval_metadata = IntervalMetadataMixin._deepcopy_(
```

Comment:
not `__deepcopy__`? See notes at the bottom [here](https://docs.python.org/3/library/copy.html#copy.deepcopy). Similar question w.r.t. `__copy__` below